core/app/models/spree/order_cancellations.rb

```diff
@@ -42,6 +42,36 @@ def short_ship(inventory_units, whodunnit:nil)
     unit_cancels
   end
 
+  def cancel_unit(inventory_unit, reason: Spree::UnitCancel::DEFAULT_REASON, whodunnit:nil)
```

Comment:
Yeah, I definitely considered that when building this. If we can reach a consensus on this change, I'd be happy to refactor `short_ship` to use this and remove the private method its using altogether as well.